ui/pattern-ls2request/src/App/App.js

```diff
@@ -1,95 +1,88 @@
 import Button from '@enact/ui/Button';
 import Item from '@enact/ui/Item';
 import PropTypes from 'prop-types';
-import {Component} from 'react';
+import {useCallback, useEffect} from 'react';
 import {connect} from 'react-redux';
 
 import {getSystemSettings, setSystemSettings, setSystemSettingsSubscribed} from '../actions';
 
 import css from './App.module.less';
 
-class App extends Component {
-	static propTypes = {
-		dispatch: PropTypes.func.isRequired,
-		brightness: PropTypes.string,
-		eyeComfortMode: PropTypes.string
-	};
-
-	componentDidMount () {
-		// This LS2Request is WITHOUT subscription
+const App = ({brightness, eyeComfortMode, dispatch}) => {
+	useEffect(() => {
 		if (typeof window.PalmSystem !== 'undefined') {
-			this.props.dispatch(getSystemSettings({
+			dispatch(getSystemSettings({
 				category: 'picture',
 				key: 'brightness'
 			}));
 
 			// This LS2Request is WITH subscription
-			this.props.dispatch(getSystemSettings({
+			dispatch(getSystemSettings({
 				category: 'picture',
 				key: 'eyeComfortMode',
 				subscribe: true
 			}));
 		}
-	}
-
-	handleDecreaseBrightness = () => {
-		let brightness = Number(this.props.brightness);
-		brightness = brightness !== 0 ? brightness - 10 : brightness;
-		return this.props.dispatch(setSystemSettings({
+	}, [dispatch]);
```

Comment:
It should be 
```suggestion
	}, []);
```

Since it's` componentDidMount`.

Reply:
Because of lint warning, // eslint-disable-next-line react-hooks/exhaustive-deps  added